libcaliptra/inc/caliptra_types.h

```diff
@@ -324,6 +324,13 @@ struct caliptra_get_idev_mldsa87_csr_resp
     uint8_t data[9216];
 };
 
+struct caliptra_get_fmc_alias_csr_resp
+{
+    struct caliptra_resp_header hdr;
+    uint32_t data_size;
+    uint8_t data[512];
```

Comment:
I think this needs to be quite a bit bigger to handle the ML-DSA CSR. The Rust definition sets it to 9216 bytes

Reply:
Looks like this is the ECC response?